### PR TITLE
feat(query): added "EKS Cluster Encryption Disabled" for Terraform

### PR DESCRIPTION
**Pull request by rafaela-soares**
**Proposed Changes**
- Added "EKS Cluster Encryption Disabled" for Terraform

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/aws/eks_cluster_encryption_disabled/query.rego [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_eks_cluster[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "'encryption_config' is defined and not null",
+		"keyActualValue": "'encryption_config' is undefined or null",
+		"searchLine": common_lib.build_search_line(["resource", "aws_eks_cluster", name], []),
+	}
+}
```
Comment (felipe-avelar) [inline assets/queries/terraform/aws/eks_cluster_encryption_disabled/query.rego:18]: I think we should add a verification that checks `resources` list contains `'secret'` as one of its values

**Review by felipe-avelar — CHANGES_REQUESTED** [review]
Please, check comments

**Review by JoaoDuarteGomes — DISMISSED** [review]
LGTM.

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Pull request merged**